R/frequency_encoder.R

```diff
@@ -16,7 +21,8 @@
 #' my_test,
 #' cat_columns = c("foo"))
 frequency_encoder <- function(X_train, X_test, cat_columns) {
-
```

Comment:
Do not forget to write a defensive test, like check that X_train and X_test are tibbles or data.frames and cat_columns is a list.